src/models/logic/orderontransfer.ts

```diff
@@ -0,0 +1,28 @@
+import { OrderOnTransfer } from "codechain-sdk/lib/core/classes";
+import models from "..";
+import * as Exception from "../../exception";
+import { OrderOnTransferInstance } from "../orderontransfer";
+import { createOrder } from "./order";
+
+export async function createOrderOnTransfer(
+    transactionHash: string,
+    orderOnTransfer: OrderOnTransfer,
+    networkId: string
+): Promise<OrderOnTransferInstance> {
+    let orderOnTransferInstance: OrderOnTransferInstance;
+    try {
+        orderOnTransferInstance = await models.OrderOnTransfer.create({
+            transactionHash,
+            spentQuantity: orderOnTransfer.spentQuantity.value.toString(10),
+            inputIndices: orderOnTransfer.inputIndices,
+            outputIndices: orderOnTransfer.outputIndices
+        });
+
+        await createOrder(transactionHash, orderOnTransfer.order, networkId);
```

Comment:
Should we create a new order whenever the order is used in a transaction?
I know that an order could be used in many transactions, I worry that duplicate orders will be stored in the DB.

Reply:
As you worried, it is problematic. This usage contradicts to the purpose of the introduction of OrderOnTransfer. I need more thinking then in what timing the order should be created.

Reply:
Now the code check if order is already exists on DB.